vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -123,6 +123,7 @@
       <AdditionalOptions>/await %(AdditionalOptions) /bigobj</AdditionalOptions>
       <AdditionalIncludeDirectories>
         $(ReactNativeWindowsDir)Microsoft.ReactNative;
+        $(ReactNativeWindowsDir)Microsoft.ReactNative\Pch;
```

Comment:
>$(ReactNativeWindowsDir)Microsoft.ReactNative\Pch; [](start = 8, length = 50)

For my education, how does this work? Since you're not not changing any of the 
#include "pch.h"
directives in the project, I assume that despite the double quotes, the pch.h file is found via the include path?

Reply:
The problem is that when we use a source file in a different project it may still reference the pch.h from the Microsoft.ReactNative project because the pch.h is at the same location as the source file. The <...> may change the order of how the include locations are probed, but it should still allow to pick up pch.h from the same location as the .cpp file. With moving pch.h file into a subfolder we rely only on the AdditionalIncludeDirectories for the pch.h to be found. If Desktop project references a .cpp file from the Microsoft.ReactNative project, the .cpp file will never be able to pick up the pch.h file because the Desktop project will not have the AdditionalIncludeDirectories with the Pch folder.